x-pack/legacy/plugins/monitoring/public/directives/main/index.js

```diff
@@ -118,6 +119,19 @@ export class MonitoringMainController {
   isMlSupported()  {
     return this._licenseService.mlIsSupported();
   }
+
+  isDisabledTab(product, _tab) {
```

Comment:
Looks like we don't need `_tab` here, and we can also remove this parameter from views when calling the function